modules/renderer/features.js

```diff
@@ -194,9 +201,7 @@ export function rendererFeatures(context) {
     });
 
     defineRule('aerialways', function isAerialways(tags) {
-        return tags.aerialway &&
-            tags.aerialway !== 'yes' &&
-            tags.aerialway !== 'station';
+        return tags?.aerialway !== 'yes' && tags?.aerialway !== 'station';
```

Comment:
This has changed the logic, if `tags = {}`, then the rule now returns `true`. If the goal is to keep typescript happy, then this would work:

```suggestion
        return !!tags?.aerialway && tags.aerialway !== 'yes' && tags.aerialway !== 'station';
```